v1/scripts/publish_prod.sh

```diff
@@ -19,15 +19,26 @@ fi
 #Read the current version
 CURRENT_VERSION=$(node -pe "require('./package.json').version")
 
-#Read the desired version
+#Read the desired version for the github release
 if [ -z "$1" ]; then
-    echo "Must specify a desired version number"
+    echo "Must specify a desired github release version number"
     exit 1
 elif [[ ! $1 =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
     echo "Must use a semantic version, e.g., 3.1.4"
     exit 1
 else
-    VERSION=$1
+    GITHUB_VERSION=$1
+fi
+
+#Read the desired version for the package release
+if [ -z "$2" ]; then
+    echo "Must specify a desired package release version number"
+    exit 1
+elif [[ ! $2 =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+    echo "Must use a semantic version, e.g., 3.1.4"
+    exit 1
+else
+    PACKAGE_VERSION=$1
```

Comment:
Previously, npm and pypi packages were published with the same version number as the github release. Now there are separate command arguments for package and github release version numbers.